server/models/executor.js

```diff
@@ -1,14 +1,20 @@
 var async = require('async');
+var crypto = require('crypto');
+var observerHelper = require('./observerHelper');
 
 module.exports = function(Executor) {
+  Executor.definition.properties.token.default = function() {
+    return crypto.randomBytes(24).toString('hex');
```

Comment:
https://nodejs.org/api/crypto.html#crypto_crypto_randombytes_size_callback

That's sync, it would be better to use a callback instead if that is possible here (it looks like it may not be).